### PR TITLE
Adds `(Host)` and `(Points to)` to the CNAME DNS editor

### PR DESCRIPTION
**Pull request by olaseni**
#### Changes proposed in this Pull Request

Improves the DNS editor when editing/creating CNAME records by adding `(Host)` after the `Name` label and `(Points to)` after `Alias of` label. This makes it easier for users that are already used to these descriptions at other providers.

#### Testing instructions

- Checkout this branch
- Navigate to the DNS editor for a domain
- Confirm the before and after images.

| Before | After |
| -------|--------|
| Screenshot 2020-10-30 at 5 05 16 PM | Screenshot 2020-10-30 at 5 04 35 PM |

### CODE REVIEW

File: client/my-sites/domains/domain-management/dns/cname-record.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ class CnameRecord extends React.Component {
 		return (
 			<div className={ classes }>
 				<FormFieldset>
-					<FormLabel>{ translate( 'Name', { context: 'Dns Record' } ) }</FormLabel>
+					<FormLabel>{ translate( 'Name (Host)', { context: 'Dns Record' } ) }</FormLabel>
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/dns/cname-record.jsx:36]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Host name' )` *ES Score: 6*


:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.

File: client/my-sites/domains/domain-management/dns/cname-record.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ class CnameRecord extends React.Component {
 				</FormFieldset>
 
 				<FormFieldset>
-					<FormLabel>{ translate( 'Alias Of' ) }</FormLabel>
+					<FormLabel>{ translate( 'Alias Of (Points to)' ) }</FormLabel>
```
Comment (stephanethomas) [inline client/my-sites/domains/domain-management/dns/cname-record.jsx:52]: Shouldn't we capitalize `to`?

```javascript
<FormLabel>{ translate( 'Alias Of (Points To)' ) }</FormLabel>
```

**Review by stephanethomas — APPROVED** [review]
I think we're missing an uppercase letter but otherwise this looks good to me. Could add the necessary labels to this pull request?

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5101393

Thank you @olaseni for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5101393).

**Pull request merged**